aidb/query/query.py

```diff
@@ -333,24 +333,72 @@ def tables_in_filtering_predicates(self):
     return tables_required_predicates
 
 
-  @cached_property
-  def columns_in_query(self):
-    """
-    nested queries are not supported for the time being
-    * is supported
-    """
+  def _get_columns_in_expression_tree(self, exp_tree):
+    '''
+    Return column_set of columns in query and
+    ordered list of columns, expression wise
+    eg: SELECT AVG(col1), COUNT(*) FROM table1;
+        column_set = {col1, col2, col3, col4}
+        exp_wise_columns = [[col1], [col1, col2, col3, col4]]
```

Comment:
why do you need to extract columns in select expression? I don't think it's necessary.

Reply:
column set is necessary for the property columns_in_query ..

Reply:
Previous implementation supports that. I mean the modification is not necessary.